examples/tutorials/navmesh_tutorial.py

```diff
@@ -0,0 +1,230 @@
+# [setup]
+import math
+import os
+import random
+
+import cv2
+import magnum as mn
+import numpy as np
+from matplotlib import pyplot as plt
+
+import habitat_sim
+from habitat_sim.utils import common as ut
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+data_path = os.path.join(dir_path, "../../data")
+output_path = os.path.join(dir_path, "navmesh_tutorial_output/")
+
+save_index = 0
+
+
+def make_video_cv2(observations, prefix="", open_vid=True, multi_obs=False):
```

Comment:
As you mentioned, it might be wise to omit this tutorial from this PR since Aaron is refactoring here.